### ISSUE TITLE
Redundant Title Text 

### ISSUE THREAD

**Issue by spencercheng** [BYU-ODH/wp-humanities-byu#464]
The `Title` tag should primarily be used for images and should be unique.

- [x] All Tags under Blogs Page (news, digital humanities, etc.) not the ones on the homepage.

**Comment by spencercheng** [comment]
Title tags are now not being used with the Tags under each post.

**Comment by WorldsEndless** [comment]
Fixed in #479
